saleor/order/models.py

```diff
@@ -146,10 +146,8 @@ def get_last_payment_status_display(self):
     def is_pre_authorized(self):
         return self.payments.filter(status=PaymentStatus.PREAUTH).exists()
 
-    def create_history_entry(self, comment='', status=None, user=None):
-        if not status:
-            status = self.status
-        self.history.create(status=status, comment=comment, user=user)
+    def create_history_entry(self, content='', user=None):
```

Comment:
We could get rid of this default value for `content` if we want to ensure that it always has a meaningful message.

Reply:
Sure.